components/Globe.vue

```diff
@@ -0,0 +1,429 @@
+  const geometry = WHONationalBorders.features.find(f => f.id === countryIso)?.geometry as MultiPolygon;
+  const bounds = am5map.getGeoBounds(geometry);
+  // Don't zoom in very tightly on the country's bounds.
+  // Also avoid exceeding the globe's bounds.
+  bounds.left = Math.max(-180, bounds.left -= 10);
+  bounds.right = Math.min(180, bounds.right += 10);
+  bounds.top = Math.min(90, bounds.top += 10);
+  bounds.bottom = Math.max(-90, bounds.bottom -= 10);
+  if (chart && centroid) {
+    pauseAnimations();
+    chart.zoomToGeoBounds(bounds, geoPointZoomDuration);
+  }
+};
+
+// Reset the globe to slowly spinning.
+const createRotateAnimation = () => {
+  const currentRotationX = chart.get("rotationX") || southEastAsiaXCoordinate;
+  return chart.animate({
+    key: "rotationX",
+    from: currentRotationX,
+    to: 360 - currentRotationX,
+    duration: 180000,
+    loops: Infinity,
+  });
+};
+
+const initializeSeries = (settings: am5map.IMapPolygonSeriesSettings) => {
+  const series = am5map.MapPolygonSeries.new(root, settings);
+  chart.series.push(series);
+  return series;
+};
+
+const handlePolygonActive = (target: am5map.MapPolygon, prevPolygonRef: Ref<am5map.MapPolygon | undefined>) => {
+  if (prevPolygonRef.value && prevPolygonRef.value !== target) {
+    prevPolygonRef.value.set("active", false);
+  }
+  prevPolygonRef.value = target;
+};
+
+const setUpBackgroundSeries = () => {
+  backgroundSeries = initializeSeries({ ...backgroundSeriesSettings, reverseGeodata: true });
+  backgroundSeries.mapPolygons.template.setAll({ tooltipText: "{name} is not currently available", toggleKey: "active", interactive: true });
+  backgroundSeries.mapPolygons.template.on("active", (_active, target) => handlePolygonActive(target, prevBackgroundPolygon));
+};
+
+const setUpSelectableCountriesSeries = () => {
+  selectableCountriesSeries = initializeSeries({ ...selectableCountriesSeriesSettings, reverseGeodata: false });
+  selectableCountriesSeries.mapPolygons.template.setAll({
+    tooltipText: "{name}",
+    toggleKey: "active",
+    interactive: true,
+    cursorOverStyle: "pointer",
+  });
+  selectableCountriesSeries.mapPolygons.template.on("active", (_active, target) => {
+    handlePolygonActive(target, prevSelectablePolygon);
+    if (prevSelectablePolygon.value && prevSelectablePolygon.value !== target) {
+      animateSeriesColour(selectableCountriesSeries, defaultLandColour);
+    }
+    if (target?.dataItem?.get("id")) {
+      appStore.globe.tentativelySelectedCountry = target.dataItem.get("id") as string;
+    }
+  });
+  selectableCountriesSeries.mapPolygons.template.states.create("hover", { fill: hoverLandColour });
+  selectableCountriesSeries.events.on("datavalidated", async () => {
+    if (appStore.selectedCountry && !appStore.globe.tentativelySelectedCountry) {
+      appStore.globe.tentativelySelectedCountry = appStore.selectedCountry;
+      await rotateToCountry(appStore.selectedCountry);
```

Comment:
When the watch updates it explicitly checks what page we're on, and doesn't rotate if we are on the results page, so that on the results page, there isn't too much background activity.